### DISCUSSION TITLE
声明：请不要为了获取权限而活跃

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3666] [讨论]
是否给予辅助管理权限取决于是否想并且有能力帮助 PCL 社区。它是让你能更好地帮助社区的 **工具**，绝非你帮助社区的 **目的** 。我只会把它给予需要它来帮助社区的人，而非想用它来彰显自己的人。它不是一个 “成就” 或者 “奖章”，这里也不是贴吧或者积分制论坛。如果你的行为是以获取权限为根本目的，我绝对不会给你权限。
请不要以此为目的去提交 issue 或者去评论，不要认为所谓的 “活跃” 就能让你拿到权限。有些人大量提交 issue，但大部分最后都被拒绝，或者它压根没到需要开 issue 或者 PR 的程度。此类行为只会为大家和我带来额外的工作量，对社区的坏处远大于好处。不要这样，谢谢大家。

这些声明已追加进 #477。

**Comment by wuliaodexiaoluo** [comment]
emmm
~（你要不直接把我身份证写上去得了~
